egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Pegah Ghahremani
+#
+# Apache 2.0.
+
+""" This script generates egs.archive.scp and ranges.* used for multilingual setup.
+    Also this script generates outputs.*.ark and weight.*.ark, where each line
+    corresponds to language-id and weight for the same example in egs.*.scp.
+    weight.*.ark is archive of table indexed by the key of input example
```

Comment:
Also the file names are {prefix}.egs.*.scp, {prefix}.output.*.ark, {prefix}.weight.*.ark and {prefix}.ranges.*.txt . e.g. prefix is 'egs' for training data and 'valid_diagnostic' for validation data.
Are you fine with that?